### PR TITLE
add wrapper to track time

### PR DESCRIPTION
**Pull request by liamhickey**

### CODE REVIEW

File: src/Component.java [hunk 1]
```diff
@@ -3,5 +3,5 @@
 {
 	c1,
 	c2,
-	c3
+	c3;
```
Comment (acournoyer1) [inline src/Component.java:6]: Lol why did this change?

**Review by acournoyer1 — APPROVED** [review]

**Review by acournoyer1 — CHANGES_REQUESTED** [review]
If you're gonna make a wrapper can you give the components an id too while you're at it?

File: src/Component.java (hunk 1 above)
Comment (liamhickey) [reply src/Component.java:6]: oh, lol, i was looking at adding the fields there, and if you add additional stuff underneath those in the enum you have to add a colon

**Comment by liamhickey** [comment]
conflicts existed with the other PR

**Pull request merged**